evolution-chain/runtime/src/lib.rs

```diff
@@ -92,120 +97,96 @@ pub mod opaque {
 	}
 }
 
-// To learn more about runtime versioning, see:
-// https://docs.substrate.io/main-docs/build/upgrade#runtime-versioning
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-	spec_name: create_runtime_str!("laos-evolution"),
-	impl_name: create_runtime_str!("laos-evolution"),
+	spec_name: create_runtime_str!("frontier-template"),
```

Comment:
revert?

Reply:
I added a comment to explain the reason: we need polkadot.js to recognize the chain as an EVM schem based

Reply:
ah I forgot about that